MIGRATION_GUIDE.md

```diff
@@ -820,7 +820,7 @@ resource "auth0_client_grant" "my_client_grant" {
 
 The Node 18 Beta runtime option for actions has been removed.
 
-Now, users will be opt-in to the GA version of Node 18 actions runtime.
+Now, you can opt in to the GA version of Node 18 actions runtime instead.
```

Comment:
This change suggests that there is a choice but in fact they are opt-in by default.